test/azure/Expected/AcceptanceTests/Paging/paging/operations/_paging_operations.py

```diff
@@ -356,6 +356,7 @@ def prepare_request(next_link=None):
                 request = self._client.get(url, query_parameters, header_parameters)
             else:
                 url = '/paging/multiple/nextOperationWithQueryParams'
+                
```

Comment:
adds a whitespace. To get around adding the whitespace, I'd have to do a check for whether the next operation has any header params that are not the `Accept` header. I thought that was a little messy, and I actually like having an extra whitespace here between the url and construct parameters, so did not add that check